File: BLL/Models/ForumThreadModel.cs
namespace Services.Models
{
    public class ForumThreadModel : BaseModel
    {
        public int? ThemeId { get; set; }
        public string ThemeName { get; set; }
        public string Title { get; set; }
        public DateTime TimeCreated { get; set; }
        public string Content { get; set; }
        public int? AuthorId { get; set; }
        public string AuthorEmail { get; set; }
        public string AuthorNickname { get; set; }
    }
}

File: ForumFrontend/src/app/models/forum-thread.model.ts
export class ForumThread {
    id: number=0;
    themeName: string='';
    title: string='';
    timeCreated: Date = new Date();
    content: string='';
    authorId: number=0;
    authorEmail: string='';
    authorNickname:string='';
    threadPostsIds: number[]=[];
    themeId: number=0;
}
